pkg/controller/directvolumemigrationprogress/directvolumemigrationprogress_controller.go

```diff
@@ -334,21 +348,19 @@ func (r *RsyncPodProgressTask) getRsyncClientContainerStatus(podRef *kapi.Pod) (
 		}
 	}
 	if containerStatus == nil {
-		if podRef.Status.Phase == kapi.PodFailed {
-			rsyncPodStatus.PodPhase = kapi.PodFailed
-			rsyncPodStatus.LogMessage = podRef.Status.Message
-			rsyncPodStatus.ContainerElapsedTime = nil
-			return &rsyncPodStatus, nil
-		}
-		return nil, fmt.Errorf("container not found")
+		log.V(4).Info("Failed to find container in Rsync Pod on source cluster",
+			"container", RsyncContainerName, "pod", path.Join(podRef.Namespace, podRef.Name))
+		return &rsyncPodStatus
 	}
 	switch {
 	case containerStatus.Ready:
 		rsyncPodStatus.PodPhase = kapi.PodRunning
 		numberOfLogLines := int64(5)
 		logMessage, err := r.getPodLogs(podRef, RsyncContainerName, &numberOfLogLines, false)
 		if err != nil {
-			return &rsyncPodStatus, err
+			log.V(4).Info("Failed to get logs from Rsync Pod on source cluster",
```

Comment:
@pranavgaikwad I should have been more specific, I think we should still log this, just not with a stacktrace.

```suggestion
			log.Info("Failed to get logs from Rsync Pod on source cluster",
```

Reply:
Done